### PR TITLE
Replace nonpublic Accumulo ColumnSet

### PR DESCRIPTION
**Pull request by SethSmucker**
ColumnSet is not part of Accumulo's public API. encode/decodeColumns are the only part of the class used in Datawave.

Replace the references of to Accumulo's ColumnSet with code that functions the same without the reference.

part of https://github.com/NationalSecurityAgency/datawave/issues/2443

### CODE REVIEW

**Review by avgAGB — DISMISSED** [review]

File: warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/reduce/AggregatingReducer.java [hunk 1]
```diff
@@ -39,6 +38,7 @@
 import com.google.common.collect.Sets;
 
 import datawave.ingest.mapreduce.job.TableConfigurationUtil;
+import datawave.util.ColumnSetUtil;
 
 @SuppressWarnings("deprecation")
```
Comment (ivakegg) [inline warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/reduce/AggregatingReducer.java]: Can we remove the suppress warnings?

**Review by ivakegg — DISMISSED** [review]

**Review by avgAGB — DISMISSED** [review]

**Review by ivakegg — APPROVED** [review]

**Review by apmoriarty — APPROVED** [review]

File: warehouse/core/src/main/java/datawave/util/ColumnSetUtil.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        encode(sb, columnFamily);
+        if (columnQualifier != null) {
+            sb.append(':');
+            encode(sb, columnQualifier);
+        }
+
+        return sb.toString();
+    }
+
+    public static Pair<Text,Text> decodeColumns(String columns) {
```
Comment (ivakegg) [inline warehouse/core/src/main/java/datawave/util/ColumnSetUtil.java:27]: Lets also avoid using Pair here (part of private api).  Use AbstractMap.SimpleEntry instead please

**Review by ivakegg — CHANGES_REQUESTED** [review]
One more update required.

**Comment by ivakegg** [comment]
Actualy, this is already handled in #2423 so I am going to close this one.  My apologies.

**Pull request closed without merging**